src/applications/accredited-representative-portal/components/POARequestDetails/POARequestDetails.jsx

```diff
@@ -0,0 +1,174 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useParams } from 'react-router-dom-v5-compat';
+import usePOARequests from '../../hooks/usePOARequests';
+import { formatDate } from '../POARequestsTable/POARequestsTable';
+
+const POARequestDetails = () => {
+  const { poaRequests } = usePOARequests();
```

Comment:
Let's update our custom hook to just have one empty state for `poaRequests`:
```javascript
const [poaRequests, setPOARequests] = useState([]);
```
And then update the other caller `POARequestsTableFetcher` so that it only considers that empty state rather than also checking for `null`.